.circleci/config.yml

```diff
@@ -6,14 +6,16 @@ orbs:
 
 # Reusable sets of steps
 commands:
-  setup_environment:
+  copy_demo_credentials:
     steps:
-      - checkout
       - run:
           name: Copy Demo SPCredentials
           command: |
             mkdir -p Simplenote/Credentials
             cp Simplenote/SPCredentials-demo.swift Simplenote/Credentials/SPCredentials.swift
```

Comment:
Rationale for having this outside `setup_environment` here: 558e932995ba7d4ef23c8e3c9c6d15247ea2b6ca